index.js

```diff
@@ -1967,6 +1986,23 @@ class GithubScm extends Scm {
                 throw err;
             });
     }
+
+    /**
+     * Returns if a user is an enterprise user
+     * @param {Object} config  The configuration object
+     * @param {String} config.token  The token used to authenticate to the SCM
+     * @param {String} config.slug   The slug of the enterprise
+     * @param {String} config.login  The login of the Github user
+     * @returns Boolean
+     */
+    async _isEnterpriseUser(config) {
+        if (!this.scmGithubGQL) {
+            return false;
+        }
+        const user = await this.scmGithubGQL.getEnterpriseUserAccount(config);
+
+        return !!user && user.type === ENTERPRISE_USER;
```

Comment:
does this verify if the enterprise user belongs, specific to yahoo enterprise account ?

Reply:
do we need to ? 🤔

Reply:
yes, it's generic and uses the configured enterprise slug for getting this data